packages/terra-form-select/src/_Frame.module.scss

```diff
@@ -249,39 +249,39 @@
 
     &.is-above {
       > .display {
-        border-bottom-left-radius: var(--terra-select-border-radius, 3px);
+        border-bottom-left-radius: var(--terra-form-select-border-bottom-left-radius, 3px);
```

Comment:
nit: I'd incoporate the is-above in the variable name. e.g. --terra-form-select-above-border-top-left-radius.

Same for the bottom border: --terra-form-select-above-border-top-right-radius.

This allows us the ability to use --terra-form-select-border-top-left-radius and --terra-form-select-border-top-right-radius with different values under the .select class if we ever need to in the future.

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/08c864fe81968bcf218d28099650960c2f45b51d#diff-a7694d0174d676146b494561fb4ffb04R252

Reply:
Resolved in this commit: https://github.com/cerner/terra-core/pull/2125/commits/08c864fe81968bcf218d28099650960c2f45b51d